packages/react-account/src/hooks/mira/use-account-mira-balance.ts

```diff
@@ -13,12 +14,12 @@ export type UseAccountMiraBalanceResult = {
 export const SCOPE_KEY_ACCOUNT_MIRA_BALANCE = ({
 	address,
 }: {
-	address: string;
+	address: Address;
 }) => ["connect-kit", "account-mira-balance", address];
 
 export function useAccountMiraBalance(): UseAccountMiraBalanceResult {
-	const account = useConnectedAccount();
-	const address = account?.address ?? "";
+	const account = useConnectedAccount()!;
+	const address = account.address!;
```

Comment:
What's the reason we need to use the non-null assertion operator here? 🤔️

Reply:
Revert